contractcourt/chain_arbitrator.go

```diff
@@ -242,6 +242,10 @@ func newActiveChannelArbitrator(channel *channeldb.OpenChannel,
 				return nil, err
 			}
 
+			if err := channel.MarkBorked(); err != nil {
```

Comment:
add comment explaining that this permanently prevents the link from being readded to the switch after it's removed